### ISSUE TITLE
How to get this running in laymans terms please?

### ISSUE THREAD

**Issue by splitn2** [cho45/GrblServer#1]
Hi @cho45 , this looks very useful and I can see an embedded system in the making with a Raspberri Pi and Arduino or something similar. Whilst not expert with these devices I generally manage to bluff my way through and making things work, to date I have been surprised and impressed with GRBL and UGS on my bench testing of the motors and electronics. Is there a way I can get this GRBLSERVER running on my MAC running Yosemite? It would be good to familiarise myself with this and compare with UGS. Cheers, Richard

**Comment by Protoneer** [comment]
Agreeeeeeed..... I would like to test it with my latest Raspberry Pi CNC board. http://blog.protoneer.co.nz/raspberry-pi-cnc-board-hat/

**Comment by splitn2** [comment]
hey @Protoneer can you please keep me posted on your hat board, I have a GAUPS shield with plug in 8825 Polulu drivers and could easily convert accross. I like the hat plug in idea to the RPi . cheers

**Comment by cho45** [comment]
GrblServer is also works on Mac OS (actually, I'm developing/debugging this on Mac).

You just connect Arduino which installed Grbl to Mac and run GrblServer.
I rewrote README.md. How is that?

**Comment by Protoneer** [comment]
hi @splitn2, Best to subscribe to me CNC board newsletter. http://eepurl.com/DIq-z (By the way I will be sending out an update in the next week or so... ;)

**Comment by splitn2** [comment]
thanks @cho45 I think thats enough for me to wing it from here and try it on my Mac. I'm running latest Yosemite and it looks like installing node.js is straight forward and I think it might already have git on the OSX out of the box otherwise I'm sure I'll find a copy to install on the mighty wwww. I've got mad cnc ideas in my head right now I am taking delivery of a large MACH3 based 3 axis router machine that will be used for snowboard construction, I have a 2nd machine half built also, both these are AC Servo driven , and another baby GRBL on the bench in testing to learn more about potential for powerful small low cost machines. Very exciting!!!!! I actually think GRBL could do a damn good job of running my larger woodworking router and it may well replace MACH3 once its developed further and faster clock speeds and faster Arduinos are accommodated? cheers Rich

**Comment by splitn2** [comment]
wow thats cool @cho45 , it took me 5 minutes after I forgot to plug my usb in first and not find my Arduino ftdi serial port device special file......... now it goes first time. Very cool. What have you got in the pipeline plans?

**Comment by ghost** [comment]
@protoneer, just dont go making claims like " we have created a Web / Browser app to allow users to remotely control their CNC machines. (Welcome to the Internet-Of-Things)"   when the screenshot clearly shows Andrew Hodel's GrblWeb with just the title changed.    Thats not cool.    Credit where credit is due.   

The hat is also based off my GrblPi range

**Comment by Protoneer** [comment]
@openhardwarecoza you are clearly as prepared as the "Bokke" was for Japan.... :) We have rewritten GRBLWeb in Python and made extra modifications like embedded Python commands in the G-Code.  

I just searched for your GRBLPi and found limited images of it (your website seem to be down)  but the bit found looks good. I do like the key pad controller you added. 

![Image of Yaktocat](https://groups.google.com/group/jscut/attach/727b7a6e2f069999/9cb771933ea92af1aff9dc47837fe26f-1069_top.png?part=0.1&authuser=0)

In all honesty my first GRBL CNC shield has been copied so many times by the Chines that I sort of see it as a complement --> [CNC Shield](http://www.aliexpress.com/item/Osoyoo-A4988-Driver-CNC-Shield-Expansion-Board-for-Arduino-V3-Engraver-3D-Printer-Free-Shipping/32464130354.html?spm=2114.01020208.3.12.MaJCHu&ws_ab_test=201556_6,201527_1_71_72_73_74_75,201409_3)

Unfortunately they are very good at copying stuff but seldom contribute to improving these projects. 

Any case, keep up the good work and do consider becoming one of the Official GRBL Sponsors like we are.
